dipy/workflow/segment.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import division, print_function, absolute_import
+
+from glob import glob
+from os.path import join, basename, splitext
+
+import nibabel as nib
+import numpy as np
+
+from dipy.segment.mask import median_otsu
+
+
+def median_otsu_bet(input_file, out_dir, save_masked=False, median_radius=4,
+                numpass=4, autocrop=False, vol_idx=None, dilate=None):
+
+    inputs = glob(input_file)
+    out_dirs = glob(out_dir)
```

Comment:
I think that if the out_dir doesn't exist we should create it.

Also the there is no need for globbing the out_dir neither zipping together with the inputs.
We zip together only different inputs or different outputs.